hooks/eslint/index.js

```diff
@@ -42,6 +44,8 @@ const installPackagesSuccess = (cwd, folderName) => {
       data.scripts.eslint =  'eslint .'
 
       return savePackageJsonIn(`${cwd}/${folderName}/package.json`, data)
+    }).catch(() => {
```

Comment:
I think add error handling is not related with this particular PR, we could open an issue for that